README.md

```diff
@@ -37,30 +39,30 @@ If you intend to pass on any additional files with configuration changes, third-
 related artifacts to the Kubernetes cluster, you may mount the desired content to `/home/wso2carbon/wso2-server-volume` directory path within
 a WSO2 product Docker container.
 
-The following example depicts how this can be achieved when passing additional configurations to WSO2 Identity Server
+The following example depicts how this can be achieved when passing additional configurations to WSO2 Identity Server nodes
```

Comment:
let's change nodes as pods, as nodes in k8s refer to host vms or bare metal machines, WDYT?

Reply:
+1